target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -225,6 +228,12 @@ ath79_setup_macs()
 		lan_mac=$(mtd_get_mac_text "mac" 4)
 		wan_mac=$(mtd_get_mac_text "mac" 24)
 		;;
+	engenius,ews511ap)
+		lan_mac=$(mtd_get_mac_text "u-boot-env" 65)
+		eth1_mac=$(macaddr_add "$lan_mac" 1)
+		ucidef_set_interface "eth0" ifname "eth0" protocol "none" macaddr "$lan_mac"
```

Comment:
A bit further down in the ath79_setup_macs() after the "esac" there's
```
[ -n "$lan_mac" ] && ucidef_set_interface_macaddr "lan" $lan_mac
```

so the lan_mac address is set twice?